pkg/codegen/internal/test/testdata/simple-enum-schema/dotnet/Enums.cs

```diff
@@ -0,0 +1,38 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Pulumi.Serialization;
+
+namespace Pulumi.PlantProvider.Enums
```

Comment:
Do you know of any issues if put enums to the main namespace? (`Pulumi.PlantProvider`) If a resource uses enums for inputs, every user's program will have to import an extra namespace, which would be great to avoid.

Reply:
Good point. I don't foresee any issues adding it to the main namespace, will proceed with that 👍